cibuildwheel/windows.py

```diff
@@ -97,6 +97,9 @@ def install_cpython(version: str, arch: str, nuget: Path) -> Path:
     nuget_args = get_nuget_args(version, arch)
     installation_path = Path(nuget_args[-1]) / (nuget_args[0] + "." + version) / "tools"
     call([nuget, "install", *nuget_args])
+    if not (installation_path / "python3.exe").exists():
+        # create python3.exe symlink, if not exists
```

Comment:
```suggestion
    # "python3" is not included in the vanilla nuget package,
    # though it can be present if modified (like on Azure).
    if not (installation_path / "python3.exe").exists():
```